docs/user/firststeps.rst

```diff
@@ -32,6 +30,26 @@ First, create a project directory:
     $ mkdir -p ~/renku-projects/myproject
     $ cd ~/renku-projects/myproject
 
+Install ``renku`` and start the shell:
+
+.. code-block:: console
+
+    $ pipenv install --pre renku
+    $ pipenv shell
+
+.. note::
+
+    You may get a ``ValueError: unknown locale: UTF-8`` - in that case, add
```

Comment:
Can we link to https://docs.pipenv.org/diagnose/#valueerror-unknown-locale-utf-8 ?

Reply:
Sure please make the change.